src/config.c

```diff
@@ -2606,6 +2602,7 @@ standardConfig configs[] = {
     createStringConfig("replica-announce-ip", "slave-announce-ip", MODIFIABLE_CONFIG, EMPTY_STRING_IS_NULL, server.slave_announce_ip, NULL, NULL, NULL),
     createStringConfig("masteruser", NULL, MODIFIABLE_CONFIG | SENSITIVE_CONFIG, EMPTY_STRING_IS_NULL, server.masteruser, NULL, NULL, NULL),
     createStringConfig("cluster-announce-ip", NULL, MODIFIABLE_CONFIG, EMPTY_STRING_IS_NULL, server.cluster_announce_ip, NULL, NULL, NULL),
+    createStringConfig("cluster-config-file", NULL, IMMUTABLE_CONFIG, ALLOW_EMPTY_STRING, server.cluster_configfile, CONFIG_DEFAULT_CLUSTER_CONFIG_FILE, NULL, NULL),
```

Comment:
Let's ditch CONFIG_DEFAULT_CLUSTER_CONFIG_FILE and just put the default value here since we're only referencing it once. You can delete CONFIG_DEFAULT_CLUSTER_CONFIG_FILE from cluster.h.

Reply:
That's better, will do